apps/publications/src/app/pages/publication-detail/publication-detail.component.ts

```diff
@@ -51,6 +53,10 @@ export class PublicationDetailComponent implements OnInit {
 
   ngOnInit(): void {
     this.initLoading = true;
+    this.disableRouting = this.disableRouting
+      ? this.disableRouting
+      : !this.authResolver.isCabinetAdmin();
```

Comment:
Wouldn't this work as well ?
```suggestion
    this.disableRouting = this.disableRouting || !this.authResolver.isCabinetAdmin();
```